coderd/apidoc/docs.go

```diff
@@ -2549,6 +2549,51 @@ const docTemplate = `{
                 }
             }
         },
+        "/organizations/{organization}/members/roles{roleName}": {
```

Comment:
missing a `/` after roles?

```suggestion
        "/organizations/{organization}/members/roles/{roleName}": {
```

Reply:
Yes! :+1: 

Typo in my swagger comment